### ISSUE TITLE
$channel->getMessageHistory() returns no result for DM channels.

### ISSUE THREAD

**Issue by key2peace** [teamreflex/DiscordPHP#362]
Putting debugs into getMessageHistory itself shows it DOES receive them, but the result is still empty.

```php
$message->channel->getMessageHistory([])->then(function ($response)
{
	dump($response);
});
```
above code works for TYPE_TEXT but not for TYPE_DM

**Comment by davidcole1340** [comment]
Related to `Channel::getGuildAttribute()` returning null, fixing now.

**Comment by davidcole1340** [comment]
Fixed with 74e5e6c6e2735a561fb47bb48a2d9ce032c0dfd1 adffb378ac8fe0e9c02ea92c894bd51a69715396
